Gemfile

```diff
@@ -53,9 +53,7 @@ group :development, :test do
   gem 'toxiproxy', '~> 0.1.3'
 end
 
-group :development do
-  gem 'rails-erd'
-end
+gem 'rails-erd', group: :development
```

Comment:
Oh! It has to do with rebase :stuck_out_tongue_closed_eyes: I had removed `quite_assets` changed group to one line but `quite_assets` was removed in different PR and I had to rebase.

I guess it saves two lines, but yes, the change is unnecessary. May be I can restore it when I update the PR for 5.0.1 if you think it is important.